sql.sql

```diff
@@ -0,0 +1,67 @@
+-- 1-Find out how many tasks are in the task table
+SELECT count(*) from task   
+-- or 
+SELECT count(id) from task
+
+-- Find out how many tasks in the task table do not have a valid due date
+
+select count(id) as WITHOUTDATE from task where due_date IS NOT NULL
+-- select  count(due_date) from task
+-- WHERE ISDATE(dt) = 1   ==>but this give me "FUNCTION hyf.ISDATE does not exist"
```

Comment:
ISDATE does not exist in MySQL, you can simply say DATE(due_date) and this will validate the date for you.